ote/src/clj/ote/services/transport.clj

```diff
@@ -541,9 +540,90 @@
+      (try
+        (email/send!
+          email
+          {:to (:user_email updated-member)
+           :subject title
+           :body [{:type "text/html;charset=utf-8"
+                   :content (str email-template/html-header
+                                 (html (email-template/notify-user-new-member updated-member requester operator title)))}]})
+        (catch Exception e
+          (log/warn "Error while sending a email to new member" e)))
+
+      ;; Create auditlog
+      (specql/insert! db ::auditlog/auditlog auditlog)
+
+      ;; Return added new-member data
+      {:id (:user_id new-member)
+       :username (:user_username new-member)
+       :name (:user_name new-member)
+       :email (:user_email new-member)})))
+
+(defn invite-new-user [email db requester operator user]
+  nil)
+
+(defn manage-adding-users-to-operator [email db requester operator-id form-data]
+
+  (let [new-member (first (fetch-user-by-email db {:email (:email form-data)}))
+        operator (first (specql/fetch db ::t-operator/transport-operator
+                                      #{::t-operator/id ::t-operator/name ::t-operator/ckan-group-id}
+                                      {::t-operator/id operator-id}))
+        operator-users (fetch-operator-users db {:operator-id operator-id})
+        new-member-is-operator-member  (some #(= (:user_id new-member) (:id %)) operator-users)
+        ;; If member exists, add it to the organization if not, invite
+        response (cond
+                        ;; Existing new-member, existing operator
+                        (and (not new-member-is-operator-member) (not (empty? new-member)) (not (empty? operator)))
+                        (add-user-to-operator email db new-member requester operator)
+
+                        ;; new new-member, existing operator -> invite new-member
+                        (and (empty? new-member) (not (empty? operator)))
+                        (invite-new-user email db requester operator (:email form-data))
+
+                        ;; new-member is already a member
+                        (and (not (empty? new-member)) new-member-is-operator-member)
+                        {:error :already-member}
+
+                        ;; Something wrong is happening
+                        :else
+                        {:error :something-went-wrong})]
+    (if (and response (not (:error response)))
+      (http/transit-response response 200)
+      (http/transit-response response 400))))
+
+(defn remove-member-from-operator [email db user operator-id member-email]
+  (let [member (first (fetch-user-by-email db {:email member-email}))
+        operator (first (specql/fetch db ::t-operator/transport-operator
+                                      #{::t-operator/id ::t-operator/name ::t-operator/ckan-group-id}
+                                      {::t-operator/id operator-id}))]
+    (specql/delete! db ::user/member
```

Comment:
What if `member` or `operator` is not found? Skip executing specql?

Reply:
Delete members from operator will be implemented in ticket: 165675263. I only made it possible in this pr to enable e2e tests. So no need to review anything related to removing member.